I'm not sure why the build failed, I don't see how this change caused it:
[2019-08-01T20:32:24.771Z] [ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.30.0:build (build-image) on project transformation-distros-docker: Unable to pull 'openjdk:11.0.3-slim-stretch' from registry 'r.ion.phx.connexta.com' : {"message":"manifest for r.ion.phx.connexta.com/openjdk:11.0.3-slim-stretch not found"} (Not Found: 404) -> [Help 1]

--- a/README.md
+++ b/README.md
@@ -35,6 +35,17 @@ ${PROTOCOL}://${LOOKUP_SERVICE_HOST}:${LOOKUP_SERVICE_PORT}/lookup
 java "-Dserver.port=8080" "-Dspring.profiles.active=dev" -jar .\target\transformation-distros-spring-<version>.jar
 ```
 
+## Push a Docker image to a registry
+
+Prerequisite: If using an insecure Docker registry, update ~/.docker/daemon.json to include the appropriate
+              insecure-registries and restart Docker.
+
+Change to the root directory of the cloned ion-transformation repository. Run the following command:
+
+```
+mvn clean install "-D-Ddocker.push.registry=myPushRegistry" docker:push
+```
+
 ## Formatting
 If during development the build fails for formatting violations:
 

--- a/distros/docker/pom.xml
+++ b/distros/docker/pom.xml
@@ -29,6 +29,7 @@
 
     <properties>
         <app.artifactId>transformation-distros-spring</app.artifactId>
+        <docker.push.registry>r.ion.phx.connexta.com</docker.push.registry>
     </properties>
 
     <dependencies>
@@ -45,9 +46,10 @@
                 <groupId>io.fabric8</groupId>
                 <artifactId>docker-maven-plugin</artifactId>
                 <configuration>
+                    <pushRegistry>${docker.push.registry}</pushRegistry>
                     <images>
                         <image>
-                            <name>r.ion.phx.connexta.com/cnxta/ion-transformation:${project.version}</name>
+                            <name>cnxta/ion-transformation:${project.version}</name>
                             <build>
                                 <dockerFile>Dockerfile</dockerFile>
                                 <assembly>

